jenkins/config/coreos-ci-lib.yaml

```diff
@@ -0,0 +1,11 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - name: "github.com/coreos/coreos-ci-lib"
```

Comment:
Shouldn't this just be e.g. `coreos-ci-lib`?

Reply:
The error I got said: `ERROR: Could not find any definition of libraries [github.com/coreos/coreos-ci-lib@main]` changing this to `github.com/coreos/coreos-ci-lib` made the pipeline move. I can try with `coreos-ci-lib` and report back what happens.

Reply:
tested with coreos-ci-lib and it also works. Updated